src/lib.rs

```diff
@@ -1293,7 +1287,7 @@ pub unsafe trait TryFromBytes {
     #[doc(hidden)] // TODO(#5): Finalize name before remove this attribute.
     fn try_read_from(bytes: &[u8]) -> Option<Self>
     where
-        Self: Sized,
+        Self: Sized + NoCell,
```

Comment:
We should have a TODO here, since we cannot claim to support `UnsafeCell`s in any meaningful sense until this bound is removed.

Reply:
done.